tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java

```diff
@@ -0,0 +1,160 @@
+package org.jboss.tools.arquillian.ui.bot.test.cruiserView;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.reddeer.common.wait.WaitUntil;
+import org.jboss.reddeer.eclipse.jdt.ui.ProjectExplorer;
+import org.jboss.reddeer.swt.api.TreeItem;
+import org.jboss.reddeer.swt.impl.browser.InternalBrowser;
+import org.jboss.reddeer.swt.impl.toolbar.DefaultToolItem;
+import org.jboss.reddeer.workbench.impl.editor.DefaultEditor;
+import org.jboss.reddeer.workbench.impl.editor.TextEditor;
+import org.jboss.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.arquillian.ui.bot.reddeer.view.ArquilliaCruiserView;
+import org.jboss.tools.central.reddeer.api.JavaScriptHelper;
+import org.jboss.tools.central.reddeer.wait.CentralIsLoaded;
+import org.jboss.tools.central.reddeer.wizards.NewProjectExamplesWizardDialogCentral;
+import org.jboss.tools.central.test.ui.reddeer.HTML5Parameterized;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Arquillia Cruiser View tests 
+ * 
+ * @author Len DiMaggio
+ *
+ */
+
+public class BasicArquilliaCruiserTest {
+
+	private static Logger log = new Logger(HTML5Parameterized.class);
```

Comment:
Changing this to BasicArquillianCruiserTest.class would make change in tests/org.jboss.tools.central.ui.bot.test/META-INF/MANIFEST.MF irrelevant.